liquibase-standard/src/main/java/liquibase/changelog/DatabaseChangeLog.java

```diff
@@ -622,7 +629,7 @@ public boolean isDbmsMatch(Set<String> dbmsSet) {
     public void includeAll(String pathName,
```

Comment:
Most `@deprecated` annotations include the method to use instead of the deprecated method. Can we add that information to this `@deprecated` as well? It might also be useful to note what should be used instead of `LabelExpression`. 

CC @MalloD12 @filipelautert

Reply:
Sure, I'll make this update.

Reply:
Thanks!